.npmrc

```diff
@@ -1,2 +1 @@
 strict-peer-dependencies=false
-save-exact=true
```

Comment:
I was requested to leave `save-exact=true` enabled, I will be looking into re-adding